test/template-tests/testcases/deploymentTemplate/Location-Should-Not-Be-Hardcoded.test.ps1

```diff
@@ -26,7 +26,7 @@ if ($locationParameter -and
     "$($locationParameter.defaultValue)".Trim() -ne 'global' -and 

```

Comment:
if we're just checking the defaultValue property here, then I think we can remove this, it should never be global.  "global" is only allowed as value on a resource.